python/ceed_vector.py

```diff
@@ -106,13 +115,26 @@ def get_array(self, memtype=MEM_HOST):
         # libCEED call
         lib.CeedVectorGetArray(self._pointer[0], memtype, array_pointer)
 
-        # Create buffer object from returned pointer
-        buff = ffi.buffer(
-            array_pointer[0],
-            ffi.sizeof("CeedScalar") *
-            length_pointer[0])
-        # Return numpy array created from buffer
-        return np.frombuffer(buff, dtype="float64")
+        # Return array created from buffer
+        if memtype == MEM_HOST:
+            # Create buffer object from returned pointer
+            buff = ffi.buffer(
+                array_pointer[0],
+                ffi.sizeof("CeedScalar") *
+                length_pointer[0])
+            # return Numpy array
+            return np.frombuffer(buff, dtype="float64")
+        else:
+            # CUDA array interface
```

Comment:
Maybe just import here? The error message would be more transparent than `NameError: name 'nbcuda' is not defined`.

Reply:
agreed